openedx/openedx-platform
# chore: remove warnings from defusedxml package

## Description

Issue: https://github.com/openedx/public-engineering/issues/146

- Refactor the deprecated `defusedxml` package which will no longer be supported.
  - The current python version used(`python3.8`) supports the `xml.etree.ElementTree` package which replace the old package (`defusedxml`). 
- Fixed the Deprecation Warnings by making this change.

## Supporting information

The best references to why it was deprecated I could find are here:
- https://github.com/tiran/defusedxml/issues/25
- https://github.com/tiran/defusedxml/issues/31

## Testing instructions

To check and find from where I get the same warning, I run the unit tests inside the `lms` container (by running `make dev.shell.lms`) and I could found that the tests which use the deprecated package are from `openedx/core/lib/tests/` path so I used `pytest` to confirm that.

Then after the changes I re-run the tests and the warning no longer appeared.

## Other information

- This change can also be the first step to deprecate/remove the `defusedxml` package since is used only in this section.
